Automattic/wp-calypso
# Update the support link for the domain connection advanced flow

#### Proposed Changes

* This fixes an incorrect link in the support documentation link for the advanced domain connection flow.

#### Testing Instructions

Visit the domain connection setup flow: domains/mapping/a8ctest.com/setup/a8c-tests.blog

Select "advanced setup"

Make sure that the link to the support documentation is: https://wordpress.com/support/domains/connect-a-domain-alternative-method/

<img width="1343" alt="Connect_a_Domain_Setup_‹_A8C_Test_Site_—_WordPress_com" src="https://user-images.githubusercontent.com/1379730/197188980-3593cc49-c0c0-4384-90a8-2f9ebe6d6050.png">


Related to https://github.com/Automattic/wp-calypso/issues/68548#issuecomment-1286426069
